packages/react-examples/src/react/DetailsList/DetailsList.Grouped.Example.tsx

```diff
@@ -118,10 +148,17 @@ export class DetailsListGroupedExample extends React.Component<{}, IDetailsListG
     );
   }
 
+  private _onRenderGroupFooter: IDetailsGroupRenderProps['onRenderFooter'] = (detailsFooterProps, defaultRender) => {
+    detailsFooterProps!.footerText = 'A footer';
```

Comment:
there a few places where I used non-null assertion operator and should have checked instead.